selene/src/json_output.rs

```diff
@@ -22,33 +22,44 @@ struct Label {
 #[derive(Serialize)]
 struct Span {
     start: usize,
+    start_line: usize,
+    start_column: usize,
     end: usize,
+    end_line: usize,
+    end_column: usize,
 }
 
-fn label_to_serializable<T>(label: &CodespanLabel<T>) -> Label {
+fn label_to_serializable(label: &CodespanLabel<codespan::FileId>, files: &codespan::Files<&str>) -> Label {
+    let start_location = files.location(label.file_id, label.range.start as u32).unwrap();

```

Comment:
Can you make both of these use `expect` with an error message? It's a lot easier to debug if something goes wrong.